lib/api/controller/document.js

```diff
@@ -523,11 +541,12 @@ class DocumentController extends NativeController {
       documents,
       { refresh, userId });
 
+    // @todo inject changes for mUpdate requests
```

Comment:
I don't understand, what changes are you talking about?

Reply:
Detailed in the PR description:

> the _updatedFields property (list of updated document keys, sent in update notifications) was only sent for document:update calls. It is now also sent for document:updateByQuery. We still need to add that prop for document:mUpdate, and that will be done in another PR (we need to modify the ES service to be able to add this feature)

Reply:
I've updated the comment to make it clearer.